test/correctness/round.cpp

```diff
@@ -25,42 +25,94 @@ bool test(Expr e, const char *funcname, int vector_width, int N, Buffer<T> &inpu
     return ok;
 }
 
-template <class T>
+template<class T>
 bool test(Expr e, const char *funcname, int N, Buffer<T> &input, T *result) {
-    return test(e, funcname, 1, N, input, result)
-        && test(e, funcname, 2, N, input, result)
-        && test(e, funcname, 4, N, input, result)
-        && test(e, funcname, 8, N, input, result);
+    return test(e, funcname, 1, N, input, result) && test(e, funcname, 2, N, input, result) && test(e, funcname, 4, N, input, result) && test(e, funcname, 8, N, input, result);
```

Comment:
Surround expression with parens and break after && to avoid long line

Reply:
Moving the binary-op to end-of-line (instead of start-of-line) is the magic incantation clang-format wants